.github/workflows/build.yml

```diff
@@ -8,7 +8,7 @@ on:
   push:
     branches:
       - 'main'
-      - '3-build-docker-image-on-push'
+      - '3-build-docker-image-on-push-2'
```

Comment:
This line should be removed before merging (calling it out just to make sure it isn't lost in the scuffle)

Reply:
Fair enough. It means that exact commit (the one following removal) cannot really be tested but pushed blind, but there is a good enough chance of success when it gets merged.